pkg/gql/query/transactions.go

```diff
@@ -92,27 +91,27 @@ func newQueryTx(tx core.ContractCall, blockHash []byte) (queryTx, error) {
 // IsNil will check for nil in a output
 func IsNil(output *core.TransactionOutput) bool {
 	if output.Pk == nil {
-		log.Warn("invalid output, Pk field is nil")
+		//log.Warn("invalid output, Pk field is nil")
```

Comment:
this logs where commented out as it was just polluting the logs with no clear added value. suggestions ?

Reply:
maybe reduce severity to `traces`?

Reply:
it does not really add anything, this logs really pollute everything when running the metrics collection.